apps/server/lib/google-drive/src/client.ts

```
@@ -0,0 +1,814 @@
+	private buildHeaders({
+		options,
+		method,
+		bodyHeaders,
+		retryCount,
+	}: {
+		options: FinalRequestOptions;
+		method: HTTPMethod;
+		bodyHeaders: HeadersLike;
+		retryCount: number;
+	}): Headers {
+		let idempotencyHeaders: HeadersLike = {};
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'idempotencyHeaders' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)